src/components/Avatar/Avatar.tsx

```diff
@@ -177,18 +125,14 @@ class Avatar extends UIComponent<Extendable<IAvatarProps>, any> {
             title={name}
           />
         )}
-        {status && (
-          <div className={classes.presenceIndicatorWrapper}>
-            <Icon
-              styles={{ root: styles.presenceIndicator }}
-              size={size < 4 ? 'micro' : size < 6 ? 'mini' : 'tiny'}
-              circular
-              name={icon}
-              variables={iconVariables}
-              title={status}
-            />
-          </div>
-        )}
+        {status &&
+          StatusIndicator.create(status, {
```

Comment:
There is no need to short circuit if `status` does not exist.  The factory will safely handle falsy values.  The `status && ` syntax can be removed.